target/riscv/cheri-archspecific-early.h

```diff
@@ -34,7 +34,16 @@
 
 #include "cheri_defs.h"
 
-// Table 3.3: Capability Exception Codes
+/*
+ * This controls how many tags are fetched with cgettags/csettags
+ */
+#ifdef TARGET_RISCV32
+#define CAP_TAG_GET_MANY_SHFT 3
```

Comment:
Maybe this would be nicer as `64 / CHERI_CAP_SIZE`?

Reply:
wouldn't that be `8`?

Reply:
:( right we don't have log2 versions of things, guess this is fine then